static/.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+**/*.js
```

Comment:
curious what the reason is to skip js files?

Reply:
There are just too many lint to fix for the legacy js code. Add comments. Hopefully while we remove the old gni and scatter codes it will get much better.